package.json

```diff
@@ -12,9 +12,11 @@
   "scripts": {
     "test": "jest",
     "coverage": "jest --coverage",
-    "build": "webpack -d --watch",
-    "server": "nodemon server/index.js",
-    "seed": "node server/index.js",
+    "build": "weback -d",
+    "build-dev:": "webpack -d --watch",
+    "start": "nodemon server/index.js",
+    "server": "node server",
+    "seed": "node db/seedScript.js",
     "schema": "mysql -uroot < db/schema.sql"
   },
```

Comment:
you should probably make use of the helpful start script, it's helpful :)

Reply:
Good point, will swap that.